update readme & remove settings.json file (#4)
Azure-Samples/functions-quickstart-javascript-azd: README.md

2) Test the HTTP GET trigger using the browser to open http://localhost:7071/api/httpGetFunction

3) Test the HTTP POST trigger in a new terminal window with the following command, or use your favorite REST client, e.g. [RestClient in VS Code](https://marketplace.visualstudio.com/items?itemName=humao.rest-client), PostMan, curl. `test.http` has been provided to run this quickly.

```bash
curl -i -X POST http://localhost:7071/api/httppostbodyfunction -H "Content-Type: text/json" --data-binary "@src/functions/testdata.json"
```

### Using Visual Studio Code
1) Open this folder in a new terminal
2) Open VS Code by entering `code .` in the terminal

> 3) Make sure the Azure Functions extension is installed

Reviewer: ```suggestion
3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
```

> 4) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete

Reviewer: Same comment as other PR - might need to teach folks how to open this.

Author: done

> 4) Add needed files to the **.vscode** folder by opening the command palette (`Crtl+Shift+P/Cmd+Shift+P` and selecting *"Azure Functions: Initialize project for use with VS Code"*

Reviewer 2: This is ok, but the following would be slightly clearer formatting if you like:
```suggestion
4) Add required files to the `.vscode` folder by opening the command palette using `Crtl+Shift+P` (or `Cmd+Shift+P` on Mac) and selecting *"Azure Functions: Initialize project for use with VS Code"*
```
